instrumentation-api/src/test/groovy/io/opentelemetry/instrumentation/api/tracer/HttpClientTracerTest.groovy

```diff
@@ -5,28 +5,29 @@
 
 package io.opentelemetry.instrumentation.api.tracer
 
-import static io.opentelemetry.auto.test.utils.ConfigUtils.updateConfig
-
+import io.opentelemetry.auto.test.utils.ConfigUtils
 import io.opentelemetry.context.propagation.TextMapPropagator
 import io.opentelemetry.instrumentation.api.decorator.HttpStatusConverter
 import io.opentelemetry.trace.Span
 import io.opentelemetry.trace.attributes.SemanticAttributes
 import spock.lang.Shared
 
 class HttpClientTracerTest extends BaseTracerTest {
-  static {
-    updateConfig {
-      System.setProperty(
-        "otel.endpoint.peer.service.mapping",
-        "myhost-mapped=reservation-service")
-    }
+  static final PREVIOUS_CONFIG = ConfigUtils.updateConfig {
+    it.setProperty(
+      "otel.endpoint.peer.service.mapping",
+      "1.2.3.4=catservice,dogs.com=dogsservice,opentelemetry.io=specservice")
+  }
```

Comment:
```suggestion
  static final PREVIOUS_CONFIG = ConfigUtils.updateConfig {
    it.setProperty(
      "otel.endpoint.peer.service.mapping",
      "dogs.com=dogsservice")
  }
```

Reply:
I can only remove `specservice`, `catservice` is used by e.g. `NetPeerUtilsTest`. Will be fixed after next PR (`ConfigValue`!)